controlpanel/frontend/jinja2/reset.html

```diff
@@ -22,12 +22,12 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
         </div>
         {% endif %}
         <h1 class="govuk-heading-l">Home Directory Reset</h1>
-        <p>Even the best curated home directories can get into a bad
+        <p>Even the best curated home directories can get into an unusable
         state (for example, a broken conda environment or Jupyter and RStudio won't
         start).</p>
 
-        <p>Reset your home directory to recreate your conda environment and shell
-        settings to a clean state. This feature automates the
+        <p>Reset your home directory to restore your conda environment and shell
+        settings to their original state. This feature automates the
         <a href="https://github.com/ministryofjustice/analytics-platform/wiki/Reset-your-R-Studio-environment">process described here</a>.</p>
 
         <div class="govuk-warning-text">
```

Comment:
I think we can remove this warning, as its not accurate. As files are stored in EFS then resetting their home directory wont delete anything - the only thing that could be lost is if a file hasnt been saved, so either we remove it entirely or update to something along the lines of "Make sure you have saved any open files before"